src/main/kotlin/me/partlysanestudios/partlysaneskies/utils/StringUtils.kt

```diff
@@ -273,6 +273,38 @@ object StringUtils {
         }
     }
 
+    fun Char.romanCharToInt(): Int {
```

Comment:
change it to 
```kotlin
fun Char.romanCharToInt() = when (this) {...
```

Reply:
there is no reason for thiss, this is how bugs get made

Reply:
same reason i dont do inline if statements

Reply:
literally nothing changes, it looks cleaner this way

Reply:
it's less clear that it's a function as opposed to a variable or other, clarity tops aesthetics in this case

Reply:
`fun Char.romanCharToInt()` literally looks functiony enough, I dont see your problem